core/datastore/src/main/kotlin/com/tripmate/android/core/datastore/OnboardingDataSourceImpl.kt

```diff
@@ -0,0 +1,30 @@
+package com.tripmate.android.core.datastore
+
+import androidx.datastore.core.DataStore
+import androidx.datastore.preferences.core.Preferences
+import androidx.datastore.preferences.core.booleanPreferencesKey
+import androidx.datastore.preferences.core.edit
+import androidx.datastore.preferences.core.emptyPreferences
+import com.tripmate.android.core.datastore.di.OnboardingDataStore
+import kotlinx.coroutines.flow.catch
+import kotlinx.coroutines.flow.first
+import java.io.IOException
+import javax.inject.Inject
+
+class OnboardingDataSourceImpl @Inject constructor(
+    @OnboardingDataStore private val dataStore: DataStore<Preferences>,
+) : OnboardingDataSource {
+    private companion object {
+        private val KEY_Personalization_COMPLETE = booleanPreferencesKey("Personalization_complete")
```

Comment:
상수는 대문자로 네이밍하는 것이 더 나을 것 같습니다. 

```suggestion
        private val KEY_PERSONALIZATION_COMPLETE = booleanPreferencesKey("personalization_complete")
```

Reply:
이런 휴먼에러를.. 감사합니다